stream-chat-android-ui-common/src/main/res/layout/stream_exomedia_default_controls_leanback.xml

```diff
@@ -76,7 +76,6 @@
         <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="@color/stream_color_primary"
```

Comment:
This file is unused anyway, doesn't matter if we remove this attribute (will clean it up more in a follow-up PR)